python/vrp.py

```diff
@@ -0,0 +1,94 @@
+from mage.geography import create_distance_matrix, LATITUDE, LONGITUDE
+from mage.constraint_programming import VRPConstraintProgrammingSolver
+from typing import Dict, List
+
+import mgp
+
+
+__distance_matrix = None
+__depot_index = None
+
+
+def get_distance_matrix(vertices):
+    """
+    Assigns distance matrix global object or returns if its already there.
+    """
+    global __distance_matrix
+
+    if __distance_matrix is not None:
+        return __distance_matrix
+
+    vertex_positions: List[Dict[str, float]] = []
+    for vertex in vertices:
+        vertex_positions.append(
+            {
+                LATITUDE: vertex.properties.get(LATITUDE),
```

Comment:
What happens if there is no property named `latitude`?

Reply:
I will handle the exception in next commit.